src/polyswarmd/artifacts/artifacts.py

```diff
@@ -0,0 +1,89 @@
+import logging
+
+from flask import current_app as app, g, Blueprint, request
+
+from polyswarmd.response import success, failure
+from polyswarmd.artifacts.client import ArtifactServiceException
+
+logger = logging.getLogger(__name__)
+artifacts = Blueprint('artifacts', __name__)
+
+# 100MB limit
+# TODO: Should this be configurable in config file?
+MAX_ARTIFACT_SIZE_REGULAR = 32 * 1024 * 1024
+MAX_ARTIFACT_SIZE_ANONYMOUS = 10 * 1024 * 1024
```

Comment:
@Rizato can we move these into the config file? Or even into kube's chart?

Reply:
@mrtizmoatwork the long term ideal plan would be for portal and polyswarmd pull it from the same place, like consul maybe.

Reply:
We should, but I would prefer to do that in a separate PR.